job-server/src/spark.jobserver/WebApi.scala

```diff
@@ -112,6 +124,12 @@ class WebApi(system: ActorSystem,
   val DefaultJobLimit = 50
   val StatusKey = "status"
   val ResultKey = "result"
+  val ResultChunkSize = if (config.hasPath("spark.jobserver.result-chunk-size")) {
```

Comment:
I wouldn't want to use exceptions for this, besides Try would swallow any exception and return a default including exception I don't want to mask.
How about
`Option("spark.jobserver.result-chunk-size").filter(config.hasPath).fold(100 * 1024)(config.getBytes(_).toInt)
`